docs(feat): Quickspin draft WIP (#1590)
armory/docs: content/en/armory-enterprise/quickspin/_index.md

---
title: Quick Spin 
linkTitle: "Quick Spin"
no_list: true
weight: 3
description: >
  Use the Armory Quick Spin container to quickly get an evaluation or testing instance of Spinnaker up and running.
exclude_search: true
toc_hide: true
---
{{< include "experimental-feature.html" >}}

## About Quick Spin
Quick Spin is a containerized Spinnaker instance you can use to very quickly get a local Spinnaker instance up and running. Quick Spin is a minimal Spinnaker installation that is installed with a single shell command for evaluating the Armory CD Self Hosted offering.

> To install Quick Spin use the steps provided in {{< linkWithTitle "install-quick-spin" >}} 
> To get started using Quick Spin use the Quick Spin Tutorial {{< linkWithTitle "quickstart-quick-spin" >}} .

Reviewer: ```suggestion
To install Quick Spin use the steps provided in {{< linkWithTitle "install-quick-spin" >}}. To get started using Quick Spin use the Quick Spin Tutorial {{< linkWithTitle "quickstart-quick-spin" >}} .
```

---
title: Quick Spin 
linkTitle: "Quick Spin"
no_list: true
weight: 3
description: >
  Use the Armory Quick Spin container to quickly get an evaluation or testing instance of Spinnaker up and running.
exclude_search: true
toc_hide: true
---
{{< include "experimental-feature.html" >}}

## About Quick Spin

> Quick Spin is a containerized Spinnaker instance you can use to very quickly get a local Spinnaker instance up and running. Quick Spin is a minimal Spinnaker installation that is installed with a single shell command for evaluating the Armory CD Self Hosted offering.

Reviewer: ```suggestion
Quick Spin is a containerized Armory CD instance that you install with a single shell command. You can use this minimal instance to evaluate the Armory CD Self-Hosted product.
```
What exactly does "minimal installation" mean? What features are in, what features have been omitted?

Question: Does Quick Spin install Spinnaker or Armory CD ? If it's the latter, what about not using "Spinnaker" at all since Spinnaker and Armory CD are technically 2 different products?

> To install Quick Spin use the steps provided in {{< linkWithTitle "install-quick-spin" >}}. 

Reviewer: ```suggestion
To install Quick Spin, use the steps provided in {{< linkWithTitle "install-quick-spin" >}}. 
```

> To get started using Quick Spin use the Quick Spin Tutorial {{< linkWithTitle "quickstart-quick-spin" >}} .

Reviewer: ```suggestion
See the {{< linkWithTitle "quickstart-quick-spin" >}} to get started using Quick Spin .
```

## Quick Spin limitations

> The Quick Spin Spinnaker instance is configured for easy install and evaluation, testing or POC use. Please consider the following usage limitations:

Reviewer: ```suggestion
The Quick Spin Spinnaker instance is configured for easy install and evaluation, testing, or POC use. It has the following usage limitations:
```
